tests/robot-tests/tests/admin/bau/comments.robot

```diff
@@ -182,21 +135,3 @@ Resolve comments
 
     ${comment_2}=    user gets comment    Fixed the problem
     user clicks button    Resolve    ${comment_2}
-
-Move publication into 'Higher Review'
-    [Tags]    HappyPath
-    user clicks link    Sign off
-    user clicks button    Edit release status
-    user clicks element    id:releaseStatusForm-approvalStatus-HigherLevelReview
-
-Switch to bau1 to approve release for immediate publication
```

Comment:
Just wondering if the purpose of this was to make sure that it's possible to approve following the comment that's now been resolved?